### DISCUSSION TITLE
Quarto - Quarto Extensions

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11408] [Blog]
# Quarto - Quarto Extensions

Quarto Extensions are a powerful way to modify or extend the behavior of Quarto, and can be created and distributed by anyone. Extension types include filters, shortcodes, and custom formats.

https://quarto.org/docs/blog/posts/2022-07-25-feature-extensions/

**Comment by mcanouil** [comment]
Quarto was already fun to use, with the extension mechanism it's 🤯

**Reply by finnoh** [reply]
Thanks for re-tweeting my blog post on quarto btw!

**Comment by jmgirard** [comment]
Is there a centralized location to collect user-created Quarto extensions?

**Reply by mcanouil** [reply]
To note, I have an idea  (beyond the [Awesome Quarto list](https://github.com/mcanouil/awesome-quarto)) to have some kind of centralised collection of extensions but it implies <https://github.com/quarto-dev/quarto-cli/issues/1836> to be fixed first.

**Comment by jjallaire** [comment]
This is the closest thing right now: https://github.com/mcanouil/awesome-quarto
